docs/dataset-overviews/amip-sst-sea-ice-boundary-forcing.md

```diff
@@ -3,64 +3,71 @@
 <!--- source_id_stub="PCMDI-AMIP" -->
 # AMIP sea-surface temperature and sea-ice boundary forcing
 
-**This section is a work in progress.**
-**For a first draft, see https://github.com/PCMDI/input4MIPs_CVs/pull/146**
-
 ## Key contacts
 
-- Names: Paul Durack
+- Names: Paul J. Durack
 - Emails: durack1@llnl.gov
 
 ## Summary
 
 <!--- begin-cmip7-phases-source-ids -->
 <!--- Do not edit this section, it is automatically updated when the docs are built -->
-### Source IDs for CMIP7 phases
+### Source IDs for CMIP phases
 
-The source ID that identifies the dataset to use in the different phases of CMIP7 is given below.
+The source ID that identifies the dataset to use is given below.
 
-#### CMIP7 AR7 fast track
+#### CMIP7
 
-For the CMIP7 AR7 fast track phase of CMIP7, use data with the source ID [PCMDI-AMIP-1-1-9](https://aims2.llnl.gov/search?project=input4MIPs&versionType=all&&activeFacets=%7B%22source_id%22%3A%5B%22PCMDI-AMIP-1-1-9%22%5D%7D)
+**an issue has been identified in the PCMDI-AMIP-1-1-9 sea ice field for the dataset last month (2022-12).
+A resolution is currently being investigated, and a revised dataset `PCMDI-AMIP-1-1-10` will be made available
```

Comment:
I would shift this above the auto-generated section, given how key it is

Reply:
The auto-generated bit is something I didn't get - the content is what I was keen to update